kamon-akka-sharding/src/main/scala/kamon/akka/sharding/instrumentation/AkkaShardingInstrumentation.scala

```diff
@@ -0,0 +1,77 @@
+package akka.kamon.instrumentation
```

Comment:
I did note that the package and the directory is not the same, but I just followed the same pattern as the instrumentation in the other modules.